src/states/BeforePreroll.js

```diff
@@ -42,6 +44,8 @@ export default class BeforePreroll extends ContentState {
    * content playback is blocked by the ad plugin.
    */
   onPlay(player) {
+    const Preroll = States.getState('Preroll');
```

Comment:
If this is what rollup needs I guess you gotta do it. My only input is that putting the getState const at the top of the file makes the most sense for keeping the code tidy.

Reply:
Yeah I tried a few ways to keep it like that, but then it became impossible to `getState` them all in the correct order. So one of the `getState` calls would return `undefined` and break everything. I think we have to do it at runtime.